Makefile

```diff
@@ -7,9 +7,12 @@
 # - CIRCLECI_TOKEN
 #
 
+AWS_ACCESS_KEY_ID := $(shell aws secretsmanager get-secret-value --secret-id=aws_access_key_id | jq -r '.SecretString')
```

Comment:
Sorry, just realized this secret has json inside it. So these should be:
```
... --secret-id=ci-user-aws-keys | jq -r '.SecretString'| jq -r .access_key_id
... --secret-id=ci-user-aws-keys | jq -r '.SecretString'| jq -r .secret_key
```

Reply:
I will make these changes to the backend Makefile as well.